src/openagi/planner/task_decomposer.py

```diff
@@ -13,7 +13,8 @@
 from openagi.prompts.base import BasePrompt
 from openagi.prompts.constants import CLARIFIYING_VARS
 from openagi.prompts.task_clarification import TaskClarifier
-from openagi.prompts.task_creator import MultiAgentTaskCreator, SingleAgentTaskCreator
+from openagi.prompts.task_creator import MultiAgentTaskCreator
+from openagi.prompts.task_creator import AutoTaskCreator,SingleAgentTaskCreator
```

Comment:
In the same line we can import MultiAgentTaskCreator @tarun-aiplanet

Reply:
resolved